plugins/strict-connection-checker/src/toolboxCategories.js

```diff
@@ -0,0 +1,849 @@
+          'kind': 'block',
+          'inputs': {
+            'SUB': {
+              'shadow': {
+                'type': 'text',
+                'fields': {
+                  'TEXT': '',
+                },
+              },
+            },
+            'TEXT': {
+              'shadow': {
+                'type': 'text',
+                'fields': {
+                  'TEXT': '',
+                },
+              },
+            },
+          },
+        },
+        {
+          'type': 'text_replace',
+          'kind': 'block',
+          'inputs': {
+            'FROM': {
+              'shadow': {
+                'type': 'text',
+                'fields': {
+                  'TEXT': '',
+                },
+              },
+            },
+            'TO': {
+              'shadow': {
+                'type': 'text',
+                'fields': {
+                  'TEXT': '',
+                },
+              },
+            },
+            'TEXT': {
+              'shadow': {
+                'type': 'text',
+                'fields': {
+                  'TEXT': '',
+                },
+              },
+            },
+          },
+        },
+        {
+          'type': 'text_reverse',
+          'kind': 'block',
+          'inputs': {
+            'TEXT': {
+              'shadow': {
+                'type': 'text',
+                'fields': {
+                  'TEXT': '',
+                },
+              },
+            },
+          },
+        },
+
+        {
+          'type': 'text_print',
+          'kind': 'block',
+        },
+        {
+          'type': 'text_prompt_ext',
+          'kind': 'block',
+          'fields': {
+            'TYPE': 'TEXT',
+          },
+        },
+      ],
+    },
+    {
+      'kind': 'category',
+      'name': 'Lists',
+      'categorystyle': 'list_category',
+      'contents': [
+        {
+          'type': 'lists_create_with',
+          'kind': 'block',
+        },
+        {
+          'type': 'lists_create_with',
+          'kind': 'block',
+        },
+        {
+          'type': 'lists_repeat',
+          'kind': 'block',
+          'inputs': {
+            'NUM': {
+              'shadow': {
+                'type': 'math_number',
+                'fields': {
+                  'NUM': 5,
+                },
+              },
+            },
+          },
+        },
+        {
+          'type': 'lists_length',
+          'kind': 'block',
+        },
+        {
+          'type': 'lists_isEmpty',
+          'kind': 'block',
+        },
+        {
+          'type': 'lists_indexOf',
+          'kind': 'block',
+
+          'fields': {
+            'END': 'FIRST',
+          },
+          'inputs': {
+            'VALUE': {
+              'block': {
+                'type': 'variables_get',
+                'fields': {
+                  'VAR': {
+                    'name': 'list',
+                  },
+                },
+              },
```

Comment:
Similarly, all of the lists blocks need their variable sub-blocks removed as well :/